### PR TITLE
styles : homepage, product page, wishlist page

### PR DESCRIPTION
**Pull request by Sharmaryan**
- UI changed
- slider now get back to original position after clicking on clear button

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
good to merge aryan

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to merge

File: src/components/Login/Login.jsx [hunk 1]
```diff
@@ -39,11 +39,36 @@ export const Login = () => {
       console.log(error);
```
Comment (Himanshu-Dhawale) [inline src/components/Login/Login.jsx:39]: remove the console.log

File: src/reducer/products-reducer.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 function ProductReducer(state, action) {
   const { selfhelp, business, biography, spirtual } = state.category;
-  console.log(action.type)
```
Comment (Himanshu-Dhawale) [inline src/reducer/products-reducer.js:3]: here again

**Review by Himanshu-Dhawale — APPROVED** [review]
Code looks good and clean.
just suggested a little change

File: src/reducer/products-reducer.js (hunk 2 above)
Comment (Sharmaryan) [reply src/reducer/products-reducer.js:3]: it is removed already

File: src/components/Login/Login.jsx (hunk 1 above)
Comment (Sharmaryan) [reply src/components/Login/Login.jsx:39]: will do this one later

**Pull request merged**